### ISSUE TITLE
Feature Request: Add AWS loadbalancer resource to check port, protocol and encryption

### ISSUE THREAD

**Issue by mrcasablr** [chef/inspec#2826]
AWS supports 3 loadbalancers -

ELB Classic - Elastic load balancer
ALB - Application load balancer
NLB - Network loadbalancer

Each load balancer supports  port, protocol. ALB/ELB supports TLS offloading as well. 

This request is to add a inspec resource that allows checking for port, protocol and TLS configuration on the loadbalancer

**Comment by tpoindessous** [comment]
Hi

just for information, in awspec, there are 3 separate resources.

Thanks !

**Comment by ncs-alane** [comment]
@clintoncwolfe I think providing discrete resources for each of the load balancers types would be conducive to writing concise controls.

**Comment by krystan** [comment]
Yup the addition of ALB and NLB would be good as they are a core aws resource.
